src/controllers/authenticationController.ts

```diff
@@ -123,10 +125,11 @@ export default class AuthenticationController {
       this.hasUserAgeLimitEnabled &&
       !isOlderThan(AGE_LIMIT)(parse(spidUser.dateOfBirth), new Date())
     ) {
-      const redirectionUrl = this.getClientErrorRedirectionUrl(
-        AGE_LIMIT_ERROR_MESSAGE,
-        AGE_LIMIT_ERROR_CODE
-      );
+      // The IO App show the proper error screen if only the `errorCode`
+      // query param is provided and `errorMessage` is missing.
```

Comment:
```suggestion
      // The IO App show the proper error screen if only the `errorCode`
      // query param is provided and `errorMessage` is missing.
      // this constrain could be ignored when this PR https://github.com/pagopa/io-app/pull/3642 is merged and 
      // released in a certain app version
```
I don't know which is your approach to keep track of future change. Mine is just a proposal

Reply:
Ok, i just extend that comment with `and that version become the minimum version supported.` We can even refers this PR with the IO App PR into the description.

Reply:
Fixed in a63459f